src/styles/admin.css

```
@@ -7252,17 +7250,17 @@
   padding: var(--hsas-space-13);
   border: 1px solid #c8cdd3;
   border-radius: 6px;
-  background: var(--hsas-color-white);
+  background: var(--hsas-surface);
 }
 
 .fuma-creator-detail-sidebar__proposal > span { color: var(--hsas-color-ink-600); font-size: var(--hsas-font-11); }
 .fuma-creator-detail-sidebar__proposal strong { display: flex; align-items: center; gap: var(--hsas-space-6); color: var(--hsas-color-ink-900); font-size: var(--hsas-font-17); }
 .fuma-creator-detail-sidebar__proposal strong i { width: 7px; height: 7px; border-radius: 50%; background: currentColor; }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)